### DISCUSSION TITLE
Integrate spreadsheet -> shiny

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#570] [Q&A] [answered]
Bagaimana caranya mengintegrasi spreadsheet ke r lalu ke shiny untuk membuat dashboard secara real time(streaming), kalau bisa diberikan referensi berupa video / dokumentasi step by step

**Comment by finesaaa** [comment, marked as answer]
Dear pak @rafiff23,
Saya pernah membuat cara step-by-stepnya dan terdokumentasi dalam Medium saya, 
https://fiqey.medium.com/using-googlesheets-as-a-database-in-r-happy-marriage-372cd4fc4770
Silahkan, semoga dapat membantu 😄
